app/src/main/kotlin/org/vinaygopinath/launchchat/screens/main/MainActivity.kt

```diff
@@ -99,6 +108,16 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    private fun showPhoneNumberSelectionDialog(phoneNumbers: List<String>, onNumberSelected: (String) -> Unit) {
+        val items = phoneNumbers.toTypedArray()
+        AlertDialog.Builder(this)
+            .setTitle(R.string.dialog_title_multiple_phone_numbers)
```

Comment:
It looks like the string resource file changes may have been left out of this PR/commit - I'm getting a "Unresolved reference: dialog_title_multiple_phone_numbers" error.